prow/config/plugins.yaml

```diff
@@ -58,11 +58,13 @@ require_matching_label:
     repo: docs
     prs: true
     regexp: (^translation/)|(type/(.*)-cherry-pick$)
+    grace_period: 10s
   - missing_label: missing-translation-status
     org: pingcap
     repo: docs-cn
     prs: true
     regexp: (^translation/)|(type/(.*)-cherry-pick$)
+    grace_period: 10s
   - missing_label: missing-translation-status
     org: pingcap
     repo: docs-dm
```

Comment:
Please configure grace_period for docs-dm as well.

Reply:
Fixed.